### PR TITLE
[Spectrum] Adds note to Spectrum FTP

### PR DESCRIPTION
**Pull request by angelampcosta**

### CODE REVIEW

**Review by RebeccaTamachiro — APPROVED** [review]

File: content/spectrum/about/ftp.md [hunk 1]
```diff
@@ -84,6 +84,6 @@ For more details, refer to the [ProFTPD documentation](http://www.proftpd.org/do
 
 Unlike FTP or FTPS, enabling Spectrum for SFTP does not require extra configuration. When setting up a Spectrum application for SSH, select port 22 and TCP.
 
-## Related
+## MS IIS FTP
```
Comment (cf-matthias) [inline content/spectrum/about/ftp.md]:
```suggestion
## Microsoft Windows IIS FTP
```

File: content/spectrum/about/ftp.md [hunk 2]
```diff
@@ -84,6 +84,6 @@ For more details, refer to the [ProFTPD documentation](http://www.proftpd.org/do
...
 
-- [IIS configuration](https://docs.microsoft.com/en-us/iis/publish/using-the-ftp-service/configuring-ftp-firewall-settings-in-iis-7#step-1-configure-the-passive-port-range-for-the-ftp-service)
+By default, [MS IIS FTP](https://docs.microsoft.com/en-us/iis/publish/using-the-ftp-service/configuring-ftp-firewall-settings-in-iis-7#step-1-configure-the-passive-port-range-for-the-ftp-service) requires that the clientIP for both FTP control and data connections are the same. However, when using Spectrum, this requirement may not be met, as both connections often terminate on different servers with their own unique egress IPs. To ensure proper functionality, set `dataChannelSecurity/matchClientAddressForPasv = false`. Refer to [MS IIS FTP Official Guide](https://learn.microsoft.com/en-us/iis/configuration/system.applicationhost/sites/site/ftpserver/security/datachannelsecurity) for further details.
```
Comment (cf-matthias) [inline content/spectrum/about/ftp.md]:
```suggestion
Refer to the [Microsoft Windows IIS documentation](https://docs.microsoft.com/en-us/iis/publish/using-the-ftp-service/configuring-ftp-firewall-settings-in-iis-7#step-1-configure-the-passive-port-range-for-the-ftp-service) to configure a static data port range and external IP matching your Spectrum application.
Additionally, IIS requires that the source IP for both, FTP control and data connections are the same. However, when using Spectrum, this requirement may not be met, as both connections often terminate on different servers with their own unique egress IPs. To ensure proper functionality, also set `dataChannelSecurity/matchClientAddressForPasv = false`. Refer to [Microsoft Windows IIS FTP Official Guide](https://learn.microsoft.com/en-us/iis/configuration/system.applicationhost/sites/site/ftpserver/security/datachannelsecurity) for further details.
```

Comment (cf-matthias) [reply content/spectrum/about/ftp.md]: Just trying to make it more obvious that the general guide linked first should be followed for these items.

**Review by cf-matthias — COMMENTED** [review]
Just trying to clarify some minor items. Thanks for the quick turnaround.

File: content/spectrum/about/ftp.md [hunk 3]
```diff
@@ -84,6 +84,7 @@ For more details, refer to the [ProFTPD documentation](http://www.proftpd.org/do
 
 Unlike FTP or FTPS, enabling Spectrum for SFTP does not require extra configuration. When setting up a Spectrum application for SSH, select port 22 and TCP.
 
-## Related
+## Microsoft Windows IIS FTP
 
-- [IIS configuration](https://docs.microsoft.com/en-us/iis/publish/using-the-ftp-service/configuring-ftp-firewall-settings-in-iis-7#step-1-configure-the-passive-port-range-for-the-ftp-service)
+Refer to the [Microsoft Windows IIS documentation](https://docs.microsoft.com/en-us/iis/publish/using-the-ftp-service/configuring-ftp-firewall-settings-in-iis-7#step-1-configure-the-passive-port-range-for-the-ftp-service) to configure a static data port range and external IP matching your Spectrum application.
```
Comment (angelampcosta) [inline content/spectrum/about/ftp.md:89]:
```suggestion
Refer to the [Microsoft Windows IIS documentation](https://docs.microsoft.com/en-us/iis/publish/using-the-ftp-service/configuring-ftp-firewall-settings-in-iis-7#step-1-configure-the-passive-port-range-for-the-ftp-service) to configure a static data port range and external IP matching your Spectrum application.

```

**Pull request merged**